### PR TITLE
Adds admin files, nuget readme, and license to GHEC

### PR DESCRIPTION
**Pull request by nickfloyd**
The purpose of this is to move the "admin" like files to the generator so that things like the `.gitignore` will be considered during build and PR generation as well as allowing us to manage these thing in one spot to better enable efficiency.

### CODE REVIEW

File: stage/dotnet/dotnet-sdk-enterprise-cloud/test-pr.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (kfcampbell) [inline stage/dotnet/dotnet-sdk-enterprise-cloud/test-pr.txt:1]: What is this file? Does this need to be here?

File: stage/dotnet/dotnet-sdk-enterprise-cloud/.gitignore [hunk 2]
```diff
@@ -114,3 +114,6 @@ dotnet-tools.json
 *.pem
 coveragereport/*
 .config/
+
+# Kiota
+.kiota.log
```
Comment (kfcampbell) [inline stage/dotnet/dotnet-sdk-enterprise-cloud/.gitignore:119]:
```suggestion
.kiota.log

```

File: stage/dotnet/dotnet-sdk-enterprise-server/.gitignore [hunk 3]
```diff
@@ -114,3 +114,6 @@ dotnet-tools.json
 *.pem
 coveragereport/*
 .config/
+
+# Kiota
+.kiota.log
```
Comment (kfcampbell) [inline stage/dotnet/dotnet-sdk-enterprise-server/.gitignore:119]:
```suggestion
.kiota.log

```

File: stage/dotnet/dotnet-sdk/.gitignore [hunk 4]
```diff
@@ -114,3 +114,6 @@ dotnet-tools.json
 *.pem
 coveragereport/*
 .config/
+
+# Kiota
+.kiota.log
```
Comment (kfcampbell) [inline stage/dotnet/dotnet-sdk/.gitignore:119]:
```suggestion
.kiota.log

```

File: stage/go/go-sdk-enterprise-cloud/.vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+    "[go]": {
+        "editor.insertSpaces": false,
+        "editor.tabSize": 4,
+        "editor.formatOnSave": true
+    }
+}
```
Comment (kfcampbell) [inline stage/go/go-sdk-enterprise-cloud/.vscode/settings.json:7]:
```suggestion
}

```

File: stage/go/go-sdk-enterprise-server/.vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+    "[go]": {
+        "editor.insertSpaces": false,
+        "editor.tabSize": 4,
+        "editor.formatOnSave": true
+    }
+}
```
Comment (kfcampbell) [inline stage/go/go-sdk-enterprise-server/.vscode/settings.json:7]:
```suggestion
}

```

File: stage/go/go-sdk/.vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+    "[go]": {
+        "editor.insertSpaces": false,
+        "editor.tabSize": 4,
+        "editor.formatOnSave": true
+    }
+}
```
Comment (kfcampbell) [inline stage/go/go-sdk/.vscode/settings.json:7]:
```suggestion
}

```

File: scripts/generate-dotnet.sh [hunk 8]
```diff
@@ -56,9 +54,12 @@ else
...
-kiota generate -l csharp --ll trace -o $(pwd)/stage/dotnet/$NAMESPACE/src/GitHub -c GitHubClient -n GitHub -d $SCHEMA_FILE --ebc
+kiota generate -l csharp --ll Information -o $(pwd)/stage/dotnet/$NAMESPACE/src/GitHub -c GitHubClient -n GitHub -d $SCHEMA_FILE --ebc
 go build -o $(pwd)/post-processors/csharp/post-processor post-processors/csharp/main.go
 post-processors/csharp/post-processor $(pwd)/stage/dotnet/$NAMESPACE/src/GitHub
 cd stage/dotnet/$NAMESPACE
 
+# Validate that the generated code compiles
 dotnet build
+
+
```
Comment (kfcampbell) [inline scripts/generate-dotnet.sh:65]:
```suggestion
```

File: stage/dotnet/dotnet-sdk-enterprise-cloud/test-pr.txt (hunk 1 above)
Comment (nickfloyd) [reply stage/dotnet/dotnet-sdk-enterprise-cloud/test-pr.txt:1]: One that needs to be deleted... thx for catching that.

**Review by kfcampbell — APPROVED** [review]

**Pull request merged**